<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QWidget" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Sigmet</string>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="typeLayout">
     <item>
      <widget class="QGroupBox" name="typeGroup">
       <property name="title">
        <string notr="true"/>
       </property>
       <layout class="QHBoxLayout" name="typeGroupLayout">
        <item>
         <widget class="QRadioButton" name="significantWeather">
          <property name="text">
           <string>Significant Weather</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="tropicalCyclone">
          <property name="text">
           <string>Tropical Cyclone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="volcanicAsh">
          <property name="text">
           <string>Volcanic Ash</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="airmansWeather">
          <property name="text">
           <string>Airman's Weather</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="sortGroup">
       <property name="title">
        <string notr="true"/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="template">
          <property name="text">
           <string>Template</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="custom">
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <layout class="QHBoxLayout" name="contentLayout">
         <property name="bottomMargin">
          <number>10</number>
         </property>
        </layout>
       </item>
       <item alignment="Qt::AlignLeft|Qt::AlignTop">
        <widget class="QLabel" name="location">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>270</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">Location</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
